### PR TITLE
Makes names bigger

### PR DESCRIPTION
**Pull request by VTCobaltblood**
Increased the character limit in names of things (papers, labels, guns, funs, ) from 26 to 63. Oh god oh fuck

### CODE REVIEW

**Comment by alsoandanswer** [comment]
...are you sure this won't cause you know...

navy seals copypasta being pasted entirely using the labeling thing

**Comment by VTCobaltblood** [comment]
No, the symbol limit is 64.

**Comment by VTCobaltblood** [comment]
!review

**Comment by skull132** [comment]
how to get lucky. The name field in MySQL is a maximum of 128 characters. Which would be reached if we pulled this PR.

**Comment by VTCobaltblood** [comment]
I'm not sure what SQL-stored objects are affected by MAX_NAME_LEN? This doesn't seem to be affecting the length of character names. Unless I'm mistaken, which I often am

**Comment by skull132** [comment]
> I'm not sure what SQL-stored objects are affected by MAX_NAME_LEN? This doesn't seem to be affecting the length of character names. Unless I'm mistaken, which I often am

The name for the character is concatenated (so last name + forename) and then pushed into the DB as a singular field. The length of that field is currently 128 within the database itself. Poze pointed out, and rightly so, that this change would make the largest possible combination be 129: `MAX_NAME_LEN + MAX_LNAME_LEN + 1`, where the `1` is the empty space added to the field.

TL;DR: change MAX_NAME_LEN to be 63 or increase the size of the SQL field.

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -160,7 +160,7 @@ Will print: "/mob/living/carbon/human/death" (you can optionally embed it in a s
 #define MAX_PAPER_MESSAGE_LEN 3072
 #define MAX_BOOK_MESSAGE_LEN  9216
 #define MAX_LNAME_LEN         64
-#define MAX_NAME_LEN          26
+#define MAX_NAME_LEN          63
```
Comment (Sindorman) [inline code/__defines/misc.dm]: Issue I see with this is that at least 58(presumably 74) things in code use this define. Mainly objects like bodybag lables, revolver, pistols, chemistry lables.

63 character ~= 12 words if we assume average word is 5 characters. That is long ass name for bodybag label

My suggestion is to seperate Mob's max name define and Object's max name in two defines.  One way would be to create new define like `MAX_NAME_MOB_LEN`, and also rename `MAX_LNAME_LEN` to be similar.

Comment (VTCobaltblood) [reply code/__defines/misc.dm]: I am fully aware of how many objects use this value. Why do you care if a bodybag label name is long? The intent of this PR is _precisely_ to allow longer names of _objects._ Mainly, weapons and paperwork (which gets absolutely ridiculous when you can't append even 1 character to a request console paper's name).

**Comment by Kaedwuff** [comment]
At long last, I can create a truly horrible tajara name.

File: code/__defines/misc.dm (hunk 1 above)
Comment (Sindorman) [reply code/__defines/misc.dm]: This will allow for mobs to have large names too(of course they can already have long ass 64 character last name).

Why do you need more than 26 characters names for objects, that's around 5-6 words. Give me an example of name that you need longer than 26 characters

Comment (VTCobaltblood) [reply code/__defines/misc.dm]: I am aware of that, as is already clear from reading my reply. I don't care.

Are you aware that all papers in request consoles are already longer than 26 characters? You cannot add anything to their name, because that just cuts the name at the 26 character mark. I can also definitely see valid use cases for names of guns or circuits (which I believe use this value as well). 

26 symbols is definitely not always 5 words, *especially* in the context of things that are named. For example, "more than 26 characters names for objects" is 41 symbols. "NSS Aurora Technology Progress Report" is 37 characters. 

You also give no arguments _against_ long names of objects aside from "well, they don't need to be that long". I disagree, I do think they may be that long.

Comment (Sindorman) [reply code/__defines/misc.dm]: Alright, do you approve this @skull132 ? What is your take on this?

Comment (MarinaGryphon) [reply code/__defines/misc.dm]: I see no issue with this.

**Comment by ToasterStrudes** [comment]
Ah yes, even more descriptive and specific diona names
